### PR TITLE
apigov-23206 - Adding the team exclusion

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by cbordier** [comment]
@lbadenhop ready for review.

File: content/en/docs/manage_marketplace/set_product_visibility.md [hunk 1]
```diff
@@ -24,14 +24,16 @@ To set the visibility for members that are registered in your provider organizat
 2. Select **Products > All Products**. The product list view is displayed.
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/set_product_visibility.md:24]:
```suggestion
2. Select **All Products**. The product list view is displayed.
```

File: content/en/docs/manage_marketplace/set_product_visibility.md [hunk 2]
```diff
@@ -24,14 +24,16 @@ To set the visibility for members that are registered in your provider organizat
...
 3. Select a product to view its details.
 4. Click **Set Visibility**.
-5. Under **Platform User**, select one of the following options from the **Visible To** menu:
+5. Under **Platform Users**, select one of the following options from the **Visible To** menu:
 
     * **Everyone** - (default) make the product visible to all registered users in your provider organization.
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/set_product_visibility.md:29]:
```suggestion
    * **Everyone** - (default) all registered users in your provider organization can see the product in the Marketplace
```

File: content/en/docs/manage_marketplace/set_product_visibility.md [hunk 3]
```diff
@@ -24,14 +24,16 @@ To set the visibility for members that are registered in your provider organizat
...
-    * **Selected user** - only members of the selected teams can see the product in the Marketplace:
+    * **Selected teams** - only members of the selected teams can see the product in the Marketplace
+    * **Exclude selected teams** - only members not part of selected teams can see the product in Marketplace
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/set_product_visibility.md:31]:
```suggestion
    * **Exclude selected teams** - only members that are not part of selected teams can see the product in the Marketplace
```

File: content/en/docs/manage_marketplace/set_product_visibility.md [hunk 4]
```diff
@@ -24,14 +24,16 @@ To set the visibility for members that are registered in your provider organizat
...
+    * **None** - nobody on arketpalce will be able to see the product
```
Comment (lbadenhop) [inline content/en/docs/manage_marketplace/set_product_visibility.md:32]:
```suggestion
    * **None** - the product is not visible to anyone in the Marketplace
```

**Review by lbadenhop — CHANGES_REQUESTED** [review]
@cbordier Please see my comments. If you agree with the changes, adjust the visibility settings under "Make product available to Marketplace users.

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**